examples/imgui_impl_sdl.cpp

```diff
@@ -284,5 +284,7 @@ void ImGui_ImplSDL2_NewFrame(SDL_Window* window)
     g_Time = current_time;
 
     ImGui_ImplSDL2_UpdateMousePosAndButtons();
+#if !defined(__ANDROID__)
     ImGui_ImplSDL2_UpdateMouseCursor();
+#endif
```

Comment:
> Could you clarify why UpdateMouseCursor cannot be called on Android?
It appears `SDL_SetCursor(...)` causes error spam **SDL/Error: That operation is not supported** in logcat. The program is still functional. I'm updating the application side to use the proper config flags.

> How about the call to SDL_GetKeyboardFocus ?
This is a bit trickier. I found [this SO post](https://stackoverflow.com/questions/26863470/sdl2-input-focus) which talks about losing keyboard/window focus on X11. If I don't remove the call I lose focus to the windows on my device.

Reply:
Thanks.

- `SDL_SetCursor()`: as I understand there's no mouse cursor anyway on Android? The right fix should then to disable it on the back-end and _not_ set `ImGuiBackendFlags_HasMouseCursors`.

- I have merged the changes to avoid calling `SDL_GetKeyboardFocus ()`. Calling `SDL_GetWindowFlags(g_Window) & SDL_WINDOW_INPUT_FOCUS` focus instead. Let me know if that works!

Reply:
> SDL_SetCursor(): as I understand there's no mouse cursor anyway on Android? ...

There doesn't appear to be any issues calling `SDL_CreateCursor...`, but it doesn't want you to actually try to set it.

I fixed the `SDL/Error: That operations is not supported" spam by adding 
```c
io.ConfigFlags |= ImGuiConfigFlags_NoMouseCursorChange | ImGuiConfigFlags_NavEnableKeyboard;
```
on the application side. The keyboard probably isn't needed for the demo, but it does work with `SDL_StartTextInput` enabled in the event loop (except backspace on my device).